include/tbb/tbb_machine.h

```diff
@@ -227,7 +227,7 @@ template<> struct atomic_selector<8> {
         #include "machine/linux_intel64.h"
     #endif
 
-#elif __linux__ || __FreeBSD__ || __NetBSD__
+#elif __linux__ || __FreeBSD__ || __NetBSD__ || __OpenBSD__
```

Comment:
Is `|| __OpenBSD__` enough to build TBB on OpenBSD (and without this improvement the build fails)?

Reply:
I did not test OpenBSD, I don't have access to the system. The reason I added this check is that OpenBSD supports futexes, and the other code in the guarded headers doesn't look system-specific.